tests/data/test_types.py

```diff
@@ -43,7 +43,7 @@ def trajectory(
         pytest.skip()
     obs = np.array([obs_space.sample() for _ in range(length + 1)])
     acts = np.array([act_space.sample() for _ in range(length)])
-    infos = np.array([{i: i} for i in range(length)])
+    infos = np.array([{f"key{i}": i} for i in range(length)])
```

Comment:
This is a test fixture for trajectories.
Before the test trajectories included info dicts whose keys were `int`s . This change ensures we only have dicts with `str` typed keys since
1. I don't think the infos dicts ever have any other key type except strings.
2. The current implementation relies on jsonpickle to serialize the info dicts. Jsonpickle would convert those int keys to strings anyways:
```python
>> jsonpickle.loads(jsonpickle.dumps({1: 1}))
{'1': 1}
```
This would make the tests fail that store a trajectory, load it again and compare the two. The original would have `int` keys while the loaded one would have `str` keys.